api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/captcha/GoogleRecaptchaProperties.java

```diff
@@ -38,4 +38,14 @@
      * The google reCAPTCHA site secret.
      */
     private String secret;
+
+    /**
+     * Whether google reCAPTCHA invisible should be enabled.
+     */
+    private boolean invisible;
+
+    /**
+     * The google reCAPTCHA position (only if invisible is enabled).
+     */
+    private String position;
```

Comment:
Please document the most likely values for the position.

Reply:
Done!